cpp_coveralls/__init__.py

```diff
@@ -82,9 +82,15 @@ def run():
 
     args.service_job_id = os.environ.get('TRAVIS_JOB_ID', '')
 
+    if args.repo_token == '' and args.service_job_id == '':
+        raise ValueError("\nno coveralls.io token specified and no travis job id found\n"
+                         "see --help for examples on how to specify a token\n")
+
     if not args.no_gcov:
         coverage.run_gcov(args)
+
```

Comment:
The commit looks very good. Just a few minor comments:

* Could you remove the blank line changes here.
* Could you rebase to merge you changes into one commit?